ee_tests/src/specs/page_objects/app/header.ts

```diff
@@ -46,16 +46,19 @@ export class RecentItemsDropdown extends ui.Dropdown {
 
 
 export class Header extends ui.BaseElement {
+  mainNavBar: ui.BaseElement;
```

Comment:
how about following the convention used everywhere 
```
mainNavBar = new ui.BaseElement(this.$('alm-app-header'));

Reply:
`SpaceDashboardPage` has a different `mainNavBar` and `LandingPage` has a different `mainNavBar`. That's why I've declared it here.